tutorials/fiori-ios-scpms-starter-mission-06/fiori-ios-scpms-starter-mission-06.md

```diff
@@ -274,13 +278,15 @@ In order to let the user have interactions with the list items we have to write
 
 4. Repeat that procedure for the **Products Table View Controller** with `ProductsTableViewController` and **Customers Table View Controller** with `CustomersTableViewController`.
 
-5. Important is that you set an identity for the `original` Navigation Controller. In the `Main.storyboard`, click on the Navigation Controller which is the detail view controller for the split view controller. Change the **Identity** to `SubNavigationController`.
+    !![Menu Bar](fiori-ios-scpms-starter-mission-06-20a.png)
+
+5. Important is that you set an identity for the original **Navigation Controller**. In the `Main.storyboard`, click on the Navigation Controller which is the detail view controller for the split view controller. Change the **Identity** to `SubNavigationController`.
```

Comment:
"It is important that..."

Please stick to the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd) and don't say "click on". In this case "select the Navigation Controller" would be better.
Please add missing comma before "which".